lib/Core/Memory.cpp

```diff
@@ -28,16 +28,72 @@
 #include <cassert>
 #include <sstream>
 
-using namespace llvm;
 using namespace klee;
 
 namespace {
-  cl::opt<bool>
-  UseConstantArrays("use-constant-arrays",
-                    cl::desc("Use constant arrays instead of updates when possible (default=true)\n"),
-                    cl::init(true),
-                    cl::cat(SolvingCat));
+llvm::cl::opt<bool>
+    UseConstantArrays("use-constant-arrays",
```

Comment:
I think the previous format is better.

Reply:
`llvm`-prefix is needed to avoid name clashes.

Reply:
I meant `cl::` clauses not alligned nicely and that string break isn't particularly helpful either

Reply:
clang-format

Reply:
sure, but `clang-format` is no god,

Reply:
it's great!

Reply:
What it produced here is clearly inferior to manual formatting.